uint8 addr
string frame
uint32 timeout
---
uint8 error
